src/db.ts

```diff
@@ -480,10 +475,7 @@ export class Db {
    * @param options - Optional settings for the command
    */
   async indexInformation(name: string, options?: IndexInformationOptions): Promise<Document> {
-    return executeOperation(
-      this.client,
-      new IndexInformationOperation(this, name, resolveOptions(this, options))
-    );
+    return this.collection(name).indexInformation(resolveOptions(this, options));
```

Comment:
same about collection vs op (just commenting so if we land on something different we don't forget to do both)